chainer/functions/activation/lstm.py

```diff
@@ -176,6 +176,12 @@ def lstm(c_prev, x):
 
     These are returned as a tuple of two variables.
 
+    This function supports variable length inputs. When mini-batch size of
```

Comment:
The mini-batch size of the current input must be equal to or smaller than that of the previous one. When mini-batch size of ...